packages/govuk-frontend/src/govuk/components/pagination/pagination.yaml

```
@@ -57,7 +61,11 @@ params:
       - name: text
         type: string
         required: false
-        description: The link text to the next page. Defaults to 'Next'.
+        description: The text content of the link to the next page. Defaults to 'Next page', with 'page' being visually hidden. If `next.html` is supplied, this is ignored.
+      - name: html
+        type: string
+        required: false
+        description: The HTML content of the link to the next page. Defaults to 'Next page', with 'page' being visually hidden. If this is provided, the `next.text` option will be ignored.
```

I think I prefer how **Summary list** calls out "If `html`" rather than "If this"?

```suggestion
        description: The HTML content of the link to the next page. Defaults to 'Next page', with 'page' being visually hidden. If `html` is provided, the `text` option will be ignored.
```

---

Using the full "`next.html`" (with parent prefix) versus just "`html`" is definitely more readable but doesn't match the name in the table column—shall we go with just `html`?

(I've looked at auto parent prefixes in https://github.com/alphagov/govuk-design-system/pull/2892)